remotion/PromoVideo/PromoVideoCTA.tsx

```diff
@@ -1,6 +1,7 @@
 import { TransitionSeries, springTiming } from "@remotion/transitions";
 import { flip } from "@remotion/transitions/flip";
 import React from "react";
+import { FaArrowRight } from "react-icons/fa";
```

Comment:
Do we need to add a whole icon set?
this adds quite a lot of bundle size

Reply:
right